modulefiles/ufs_orion.intel

```diff
@@ -8,7 +8,7 @@ module-whatis "loads UFS Model prerequisites for Orion/Intel"
 
 module load contrib noaatools
 
-module load cmake/3.18.1
+module load cmake/3.22.1
```

Comment:
Is there a minimum cmake version required? Cheyenne.gnu/intel is loading cmake/3.18.2. Are changes required there?

Reply:
WW3 requires 3.19

Reply:
@DeniseWorthen is Cheyenne cmake/3.22.0  the correct version to switch to for the Cheyenne modules?  (That was the version you mentioned on the WW3 PR, I forgot to update it here).

Reply:
Cheyenne has cmake/3.22.0. Thanks.

Reply:
Updated for Cheyenne

Reply:
Just a thought, but it might be wise to update the `cmake_minimum_required` in UFS to 3.19.

Reply:
@DusanJovic-NOAA Could you make this change in your PR (3.18->3.19):
https://github.com/ufs-community/ufs-weather-model/blob/10a616d328356cf99b36961a392eac65a3359db2/CMakeLists.txt#L1

Reply:
Done.